src/lib.rs

```diff
@@ -562,9 +566,12 @@ impl Client {
     /// You can get the `access_token` at
     /// <https://rollbar.com/{your_organization}/{your_app}/settings/access_tokens>.
     pub fn new<T: Into<String>>(access_token: T, environment: T) -> Client {
-        let https = HttpsConnector::new();
-        let client = hyper::Client::builder()
-            .build::<_, Body>(https);
+        let https = HttpsConnectorBuilder::new()
+            .with_native_roots()
+            .https_only()
+            .enable_http1()
```

Comment:
Do we for sure need to configure all this? Unclear to me looking at the docs what the defaults are...